### DISCUSSION TITLE
Rename `cangen`?

### DISCUSSION THREAD

**Discussion by BlakeFreer** [macformula/racecar#305] [General]
Cangen is the name of a linux tool for generating CAN messages

https://manpages.debian.org/stretch-backports/can-utils/cangen.1.en.html

Should we rename our program to avoid conflicts on linux? Could we give it a cooler name??

@AndrewI26

**Comment by AndrewI26** [comment]
We could go with something like cancreator or cancreate. Or even something like canmake would also be cool. But not sure if canmake sounds too similar to cmake.
